### DISCUSSION TITLE
Where can I find the API documentation?

### DISCUSSION THREAD

**Discussion by intaek-h** [atlassian/pragmatic-drag-and-drop#97] [Q&A]
I have been reading through the doc that's in the Atlassian website but all I see are examples and tutorials.

I want to take a look at the API spec so I can decide which ones to use.

**Comment by alexreardon** [comment]
It can be a bit tricky to find. Have a look in the sidebar.

- Core package: https://atlassian.design/components/pragmatic-drag-and-drop/core-package
- Optional packages: https://atlassian.design/components/pragmatic-drag-and-drop/optional-packages

**Reply by intaek-h** [reply]
thank you :)
